apps/data/admin.py

```diff
@@ -1 +1,25 @@
-# Register your models here.
+from django.contrib import admin
+from .models import Entry, Repository
+
+
+@admin.register(Repository)
+class RepositoryAdmin(admin.ModelAdmin):
+    search_fields = ['name']
+    ordering = ['name']
```

Comment:
I would prefer defining default order in the model, please declare another issue for this and we will address it later.